tests/recovery.py

```diff
@@ -182,8 +183,30 @@ def run(args):
             for node in network.get_joined_nodes():
                 node.verify_certificate_validity_period()
 
+            primary, _ = network.find_primary()
+
             LOG.success("Recovery complete on all nodes")
 
+    # Verify that a new ledger chunk was created at the start of each recovery
+    ledger = ccf.ledger.Ledger(primary.remote.ledger_paths(), committed_only=False)
+    for chunk in ledger:
+        first_tx_in_chunk = True
```

Comment:
Perhaps we should add functions to get first and last seqno from the `LedgerChunk`. That would make this a little easier to read, I think.

Reply:
Done.